### ISSUE TITLE
GolangCI dead

### ISSUE THREAD

**Issue by duanehoward** [google/gonids#158]
GolangCI linter died over a year ago, move to "new" github action.

**Comment by duanehoward** [comment]
details here: https://github.com/golangci/golangci-lint-action

**Comment by duanehoward** [comment]
constantly getting
```
run golangci-lint
  Running [/home/runner/golangci-lint-1.40.1-linux-amd64/golangci-lint run --out-format=github-actions] in [] ...
  level=error msg="Running error: context loading failed: no go files to analyze"
```

with default configurations and random variants thereof.
This may be due to the lack of go module support, or something else.

**Comment by duanehoward** [comment]
fixed in #161
